packages/pageflow-react/src/loadingSpinner/components/MediaLoadingSpinner.jsx

```diff
@@ -82,12 +86,13 @@ function preventScrollBouncing(e) {
   e.preventDefault();
 }
 
-function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength}) {
+function backgroundImageInlineStyles({firstPageBackgroundImageUrl, backgroundImage, blurStrength, backgroundImageX, backgroundImageY}) {
   const url = backgroundImage ? backgroundImage.urls.medium : firstPageBackgroundImageUrl;
   if (url) {
     return {
       backgroundImage: `url("${url}")`,
-      filter: 'blur('+blurStrength+'px)'
+      filter: 'blur('+blurStrength+'px)',
+      backgroundPosition: backgroundImage ? `${backgroundImageX}% ${backgroundImageY}%`: ''
```

Comment:
What happens when background position has not been set?

Reply:
actually I had some unexpected behaviour yesterday with pageflow-react js 
while working with local and remote copy and that made me scratch my head really hard because
of unexpected behaviour and some how these commits also reflect that. I need to fix this MR